src/internal/packager/images/pull.go

```diff
@@ -69,40 +77,33 @@ func (i *ImgConfig) PullAll() error {
 	}
 	spinner.Updatef("Preparing image sources and cache for image pulling")
 
-	var (
-		progress    = make(chan v1.Update, 200)
-		progressBar *message.ProgressBar
-		title       string
-	)
+	spinner.Success()
+	title := fmt.Sprintf("Pulling %d images (%s of %d)", imgCount, "0", imgCount)
+	spinner = message.NewProgressSpinner(title)
```

Comment:
TODO: refactor this to use a progress bar, might need to fork crane to get the bytes.

Reply:
Refactored !!!